Processen/CR-PR-Haven.md

```diff
@@ -14,6 +14,17 @@ Het onderstaande diagram beschrijft hoe er wordt om gegaan met issues die in het
 
 De stappen in het Change- en Problem Management proces van Haven zijn genummerd met een uniek nummer en hieronder in meer detail beschreven.
 
-**6.1** <br/>
-**6.2** <br/>
-...
+| **Stap** | **Omschrijving** |
+| -------- | ---------------- |
+| **6.1** | Vanuit het Service Desk proces komt een wens of bugmelding binnen. |
+| **6.2** | API-Beheer beoordeelt of het gaat om een Operationeel probleem (ga naar 6.3) of om een wens (ga naar 6.12). |
+| **6.3** | API-Beheer analyseert het probleem (opnieuw). |
+| **6.4** | Is het probleem reproduceerbaar (ga naar 6.6) of niet (ga naar 6.5). |
+| **6.5** | Is er een verklaring waarom het probleem niet reproduceerbaar is (ga naar 4.12) of niet (ga naar 4.13). |
+| **6.6** | Beschrijf het probleem. |
+| **6.7** | Is het probleem zelf op te lossen (ga naar 6.8) of niet (ga naar 6.12). |
+| **6.8** | API-Beheer lost het probleem op. |
+| **6.9** | API-Beheer meldt aan de indiener van het probleem dat het probleem is opgelost en vraagt de indiener of dat naar wens is. |
+| **6.10** | Is het probleem naar wens opgelost (ga naar 6.11) of niet (ga naar 6.3). |
+| **6.11** | Het issue wordt gesloten. |
+| **6.12** | Het issue wordt doorverwezen naar Haven-beheer. |
```

Comment:
Hier zit een stapje voor.

De vraag is eerst of het een Haven "cluster" probleem is, bijvoorbeeld: heel Haven ligt plat.
Het kan namelijk een ZGW referentieimplementatie specifiek probleem zijn, bijvoorbeeld: de database bevat een migratiefout in nieuwe deployment.

Reply:
Aanvulling: in het laatst benoemde geval, is de juiste stap om het probleem bij ZGW te toetsen.

Reply:
Wordt dit niet afgedekt door het Service Desk proces (https://github.com/VNG-Realisatie/api-beheer/blob/master/Processen/ServiceDesk.md)?

Reply:
Goed punt. Akkoord :-)